persistent-mysql/test/main.hs

```diff
@@ -119,7 +119,7 @@ main = do
       , Recursive.recursiveMigrate
       , CompositeTest.compositeMigrate
       , PersistUniqueTest.migration
-      , RenameTest.migration
+      -- , RenameTest.migration
```

Comment:
Currently broken on mysql because `DEFAULT=CURRENT_DATE` *is not supported on mysql* ???